### ISSUE TITLE
Replace test site for FI.HE-02

### ISSUE THREAD

**Issue by LordArrin** [hyperion-cs/dpi-checkers#36]
{ id: "FI.HE-02", provider: "🇫🇮 Hetzner", times: 1, url: "https://nioges.com/libs/fontawesome/webfonts/fa-solid-900.woff2" },

Looks like site is dead, causes false negative results.

**Comment by hyperion-cs** [comment]
Perhaps there was a temporary downtime. I don't see any problems now:
```
[15:24:13.861] DPI checking(#FI.HE-02)/INFO: HTTP 200
[15:24:13.862] DPI checking(#FI.HE-02)/INFO: Received chunk: 7957 bytes, total: 7957
[15:24:13.867] DPI checking(#FI.HE-02)/INFO: Received chunk: 2199 bytes, total: 10156
[15:24:13.871] DPI checking(#FI.HE-02)/INFO: Received chunk: 3873 bytes, total: 14029
[15:24:13.878] DPI checking(#FI.HE-02)/INFO: Received chunk: 1291 bytes, total: 15320
[15:24:13.885] DPI checking(#FI.HE-02)/INFO: Received chunk: 1291 bytes, total: 16611
[15:24:13.894] DPI checking(#FI.HE-02)/INFO: Received chunk: 1291 bytes, total: 17902
[15:24:13.914] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 20484
[15:24:13.918] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 23066
[15:24:13.921] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 25648
[15:24:13.925] DPI checking(#FI.HE-02)/INFO: Received chunk: 3873 bytes, total: 29521
[15:24:13.931] DPI checking(#FI.HE-02)/INFO: Received chunk: 2754 bytes, total: 32275
[15:24:13.945] DPI checking(#FI.HE-02)/INFO: Received chunk: 8129 bytes, total: 40404
[15:24:13.950] DPI checking(#FI.HE-02)/INFO: Received chunk: 3490 bytes, total: 43894
[15:24:13.956] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 46476
[15:24:13.959] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 49058
[15:24:13.965] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 51640
[15:24:13.970] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 54222
[15:24:13.976] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 56804
[15:24:13.978] DPI checking(#FI.HE-02)/INFO: Received chunk: 3873 bytes, total: 60677
[15:24:13.983] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 63259
[15:24:13.989] DPI checking(#FI.HE-02)/INFO: Received chunk: 2582 bytes, total: 65841
[15:24:13.990] DPI checking(#FI.HE-02)/INFO: Early complete (622.2 ms)
```

**Comment by LordArrin** [comment]
Yep, now it's alive.
